tests/network/sriov.go

```diff
@@ -686,3 +667,24 @@ func defaultCloudInitNetworkData() string {
 	ExpectWithOffset(1, err).ToNot(HaveOccurred(), "should successfully create default cloud init network data for SRIOV")
 	return networkData
 }
+
+func findInterfaceHavingMAC(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance, mac string, timeout time.Duration) (string, error) {
+	var ifaceName string
+	err := wait.Poll(timeout, 5*time.Second, func() (done bool, err error) {
+		vmi, err := virtClient.VirtualMachineInstance(vmi.GetNamespace()).Get(vmi.GetName(), &k8smetav1.GetOptions{})
+		if err != nil && errors.IsNotFound(err) {
+			return false, err
+		} else if err != nil {
+			return false, nil // we should re-try on client errors
```

Comment:
Why do you need the comment? It says after all what is just written in code.
If you prefer, you can create constants `retry` and `done`.